Modules/socketmodule.c

```diff
@@ -8421,6 +8421,12 @@ socket_exec(PyObject *m)
 #ifdef  IP_RECVTOS
     ADD_INT_MACRO(m, IP_RECVTOS);
 #endif
+#ifdef  IP_RECVERR
```

Comment:
Can you insert these in alphabetical order?

OOC why these two options? Just looking at man 7 ip, it looks like there are a few other options we are missing, e.g. IP_RECVORIGDSTADDR. Would it make sense to add more to make this list of options more complete?

Reply:
Sure, I added these two, because I only needed these two :)
Will add more.